### PR TITLE
fix(buttons): large ButtonSizes height

### LINKED ISSUE #68: bug(button): Wrong large button height
**Describe the bug**
### Current behavior
Button has not the right size
Large is actually 64.dp 

### Expected behavior
Large Button is 56.dp 

**Screenshots**

image
image

**Smartphone (please complete the following information):**

- Device: all
- OS: all
- Version 0.2.0-alpha01

### PR DESCRIPTION
**Pull request by corentin-stamper**
## Changes description 🧑‍💻

- Updates large ButtonSizes height (64.dp to 56.dp)

## Context 🤔

Closes #68 

## Checklist ✅

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check compose contract convention. It must follow conventions described [here](/CONVENTIONS.md).
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- [x] I have tested on a tablet device/emulator.
- [x] I have tested on a large screen device/emulator.
- [x] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

### CODE REVIEW

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/3c35dbc8-298b-403a-a0e7-31a0c3ed14ca/artifact/b5d459e3379b5885/p/717600fe429eb2e0df9d87b1333587d3

**Review by ManonPolle — APPROVED** [review]

File: buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt [hunk 1]
```diff
@@ -24,7 +24,7 @@ object VitaminButtonsSizes {
         fontSize: TextStyle = VitaminTheme.typography.button
             .copy(fontSize = VitaminTheme.typography.text1.fontSize),
         minWidth: Dp = 84.dp,
-        height: Dp = 64.dp,
```
Comment (ManonPolle) [inline buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt:27]: Did you change & verify paparazzi tests ?

File: buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt [hunk 2]
```diff
@@ -24,7 +24,7 @@ object VitaminButtonsSizes {
...
+        height: Dp = 56.dp,
```
Comment (ManonPolle) [inline buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt:27]: Did you change & verify paparazzi tests ?

**Review by ManonPolle — CHANGES_REQUESTED** [review]

File: buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt (hunk 2 above)
Comment (corentin-stamper) [reply buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt:27]: Done

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/c1043e67-b8f7-4c11-a337-b6db265d9b2c/artifact/6f595da3a6ed29a4/p/6c940d5c318d794424a0e25427770b7f

**Review by ManonPolle — APPROVED** [review]

**Comment by SimonLeclercq** [comment]
👍

**Review by SimonLeclercq — APPROVED** [review]

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/314333ed-50db-49a6-b7b2-68f0ce0ce356/artifact/9a0096369bfcf2e9/p/bf220ff579a6469e009bdaa5648d47e2

**Pull request merged**